src/commands.js

```diff
@@ -354,11 +354,83 @@ module.exports = {
   },
   githubLogin: {
     method: 'POST',
-    path: '/auth/github/login',
+    path: '/auth/{{mount_point}}/login',
```

Comment:
What about a default `mount_point`? Good idea or not?

Reply:
I thought about that, but didn't have a strong opinion either way.

My understanding is that the `github` mount point is the default by convention. But supporting a default would help with backwards compatibility - so that might be a preferred approach.

Reply:
Actually, as I go back through the documentation, every authentication provider has the same comment. So if any authentication backend can be mounted at any point, this is a bit more broad change to get right.